### PR TITLE
Update Determining-the-Cost.md

### PR DESCRIPTION
**Pull request by PhilippRenner**
By fully outline the Word logarythm, a translation mistake will be fixed. "log" was translated like log in "error-log", but I think Logarythm is meant!

### CODE REVIEW

**Comment by PRMerger6** [comment]
@PhilippRenner : Thanks for your contribution! The author(s) have been notified to review your proposed change. @iainfoulds

File: WindowsServerDocs/identity/ad-ds/plan/Determining-the-Cost.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ The following table lists the speeds for different types of networks.
 |100BaseT|100 Mbps|
 |Gigabit Ethernet|1 gigabit per second (Gbps)|
 
-Use the following table to calculate the cost of each site link based on wide area network speed (WAN) link speed. For WAN link speed that is not listed in the table, you can calculate a relative cost factor by dividing 1,024 by the log of the available bandwidth, as measured in Kbps.
+Use the following table to calculate the cost of each site link based on wide area network speed (WAN) link speed. For WAN link speed that is not listed in the table, you can calculate a relative cost factor by dividing 1,024 by the logarythm  of the available bandwidth, as measured in Kbps.
```
Comment (illfated) [inline WindowsServerDocs/identity/ad-ds/plan/Determining-the-Cost.md:38]: Pretty good correction, just needs a minor change to be correct English:
```suggestion
Use the following table to calculate the cost of each site link based on wide area network speed (WAN) link speed. For WAN link speed that is not listed in the table, you can calculate a relative cost factor by dividing 1,024 by the logarithm of the available bandwidth, as measured in Kbps.
```
Thank you for catching the translation issue based on the multiple meanings of the word "log".

Comment (PhilippRenner) [reply WindowsServerDocs/identity/ad-ds/plan/Determining-the-Cost.md:38]: Okay, thanks for correcting me. What to do next to realize the changes?

**Review by eross-msft — APPROVED** [review]

**Comment by illfated** [comment]
Thank you for the PR. I was just a tad late to answer your question, but I guess it is moot now. 😉

**Pull request merged**